lib/deps.js

```diff
@@ -1,7 +1,11 @@
 'use strict';
 
+const CWD = process.cwd();
```

Comment:
Do we still not support TS application? as this will probably not work with `dist`.

Reply:
We don't support TS but TS may have different compile destination. We expect all app code separated to .js + .d.ts. TypeScript can check js files against types.